dhis-2/dhis-test-integration/src/test/java/org/hisp/dhis/dxf2/deprecated/tracker/TrackedEntityServiceTest.java

```diff
@@ -502,6 +502,34 @@ void testSavePerson()
         assertEquals( ImportStatus.SUCCESS, importSummary.getStatus() );
     }
 
+    @Test
+    void shouldCreateTeiWhenValidateUniqueAttribute()
```

Comment:
Can you write this test in https://github.com/dhis2/dhis2-core/blob/master/dhis-2/dhis-test-integration/src/test/java/org/hisp/dhis/trackedentity/TrackedEntityServiceTest.java ?

All `deprecated/tracker` code will be removed (one day). Tests of code that will not be removed with old tracker should thus be written in test classes outside of any deprecated package.

Reply:
ok @teleivo this looks difficult to bring in the test class you are mentioning. I've made a similar test for the entity attribute store

Reply:
sorry the test is not correct yet, it needs some adjustment

Reply:
it should be ok now, the test is to check attribute  value uniqueness otherwise it makes only sense in the old tracker importer